pages/careers/_open-positions.html

```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
 
     <hr />
 
-    <h3>Senior Software Engineer<br />Senior Site Reliability Engineer</h3>
-    <p>
-      Just like our <a href="#software-engineer">Software Engineer</a> and
-      <a href="#site-reliability-engineer-sre">Site Reliability Engineer</a>
-      positions, but with the added expectation that you take more ownership,
-      get (complex) things done, and communicate even more effectively.
-    </p>
+    <a target="senior-product-designer"></a>
+    <h3>Senior Product Designer</h3>
 
-    <hr />
+    <p>Help us make Gruntwork delightful!</p>
+
+    <h4>What's Gruntwork?</h4>
+    <p>We give software teams a complete, world-class cloud infrastructure that lets them go to prod in days. Typically,
+       the process of launching prod infrastructure in the cloud is awful. It takes months of tedious work, hundreds of
+       thousands of dollars, and often results in poorly maintained code and anti-patterns in the end. We call our company
+       "Gruntwork" because we take away this Gruntwork for our clients so that they can focus on the parts of their software
+       that are unique to them.</p>
+    <p>We are 100% remote, ~15 people, profitable, growing (even during COVID!) and fully bootstrapped with no debt and
+       no investors. Our small company works with both startups and name companies like Toyota, Adobe, and the US House
+      of Representatives. We also <a href="#gruntwork-is-a-human-friendly-company">care a lot about humans.</a></p>
+
+    <h4>What's the opportunity?</h4>
```

Comment:
This title should be "What you'll work on" to be consistent with our other job roles.

Reply:
Fixed in 629e840.